src/contacts/ContactImporter.ts

```diff
@@ -29,6 +29,8 @@ import { ImportNativeContactBooksDialog } from "./view/ImportNativeContactBooksD
 import { StructuredContact } from "../native/common/generatedipc/StructuredContact.js"
 import { isoDateToBirthday } from "../api/common/utils/BirthdayUtils.js"
 import { ContactBook } from "../native/common/generatedipc/ContactBook.js"
+import { requestPermission } from "../gui/dialogs/SetupWizard.js"
```

Comment:
I am unhappy that `requestPermission()` is grabbing random things from the locator and I wouldn't use it as-is. At least we should move it out of `SetupWizard` and even better would be refactor it into something injectable.